package.json

```diff
@@ -36,14 +39,25 @@
     "finalhandler": "^0.4.0",
     "github": "^0.2.3",
     "glimmer-engine": "tildeio/glimmer#d6ae47a",
-    "glob": "~4.3.2",
+    "glob": "^5.0.13",
     "htmlbars": "0.14.14",
+    "mocha": "^2.4.5",
+    "mocha-only-detector": "0.0.2",
     "qunit-extras": "^1.4.0",
     "qunitjs": "^1.20.0",
+    "rimraf": "^2.5.2",
     "route-recognizer": "0.1.5",
     "rsvp": "~3.1.0",
     "serve-static": "^1.10.0",
     "simple-dom": "^0.2.7",
     "testem": "^1.0.0-rc.3"
+  },
+  "dependencies": {
```

Comment:
I think this is something that would need to get solved on the ember-cli side more globally. If we put those dependencies into `devDependencies` instead, they usually won't get installed and might be missing when someone tries to generate blueprints.